ports/esp32s2/common-hal/busio/I2C.c

```diff
@@ -149,12 +142,14 @@ void common_hal_busio_i2c_deinit(busio_i2c_obj_t *self) {
         return;
     }
 
-    i2c_status[self->i2c_num] = STATUS_FREE;
+    i2c_driver_delete(self->i2c_num);
 
-    common_hal_reset_pin(self->sda_pin);
     common_hal_reset_pin(self->scl_pin);
+    common_hal_reset_pin(self->sda_pin);
```

Comment:
If this change of order is important, can you comment about it?

Reply:
It is not important - I can undo that to make it consistent with the ordering elsewhere.